diff --git a/node_modules/ai/rsc/dist/rsc-server.mjs b/node_modules/ai/rsc/dist/rsc-server.mjs
index 5e55d3e..e6645de 100644
--- a/node_modules/ai/rsc/dist/rsc-server.mjs
+++ b/node_modules/ai/rsc/dist/rsc-server.mjs
@@ -161,9 +161,7 @@ async function innerAction({
     }
   );
 }
-function wrapAction(action, options) {
-  return innerAction.bind(null, { action, options });
-}
+
 function createAI({
   actions,
   initialAIState,
@@ -2551,6 +2549,13 @@ function createStreamableValueImpl(initialValue) {
   };
   return streamable2;
 }
+
+function wrapAction(action, options) {
+  const ii = eval('_$$INLINE_ACTION');
+  console.log("wrapAction", ii, innerAction, action, options);
+  return ii.bind(null, { action, options });
+}
+
 export {
   createAI,
   createStreamableUI,
